src/main/java/duke/functions/TaskList.java

```diff
@@ -0,0 +1,67 @@
+package duke.functions;
+
+import duke.tasks.Task;
+
+/**
+ * Class which holds the list of tasks inputted by a user.
+ * @author lauralee
+ */
+public class TaskList {
+
+    Task[] taskArr;
+
+    /**
+     * Constructor for TaskList class.
+     */
+    public TaskList() {
+        this.taskArr = new Task[100];
+    }
+
+    /**
+     * Getter for TaskList which stores tasks inputted by the user.
+     * @return The Task array containing the tasks that has been inputted by the user.
+     */
+    public Task[] getTaskArr() {
+        return this.taskArr;
+    }
+
+    /**
+     * Marks a task as complete.
+     * @param taskPos The position of the task which the user is marking as complete in the TaskList.
+     */
```

Comment:
The Javadoc comments are clear and easy to understand and I like how you followed the coding standard for writing Javadoc comments as well!